### PR TITLE
tutorials blog added!

### PR DESCRIPTION
**Pull request by LideviK**

### CODE REVIEW

File: src/_content/posts/tutorials.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+author: Lidya K
+category: General
+published_datetime: 2024-08-18 07:56:00-04:00
+---
+## We’re thrilled to announce the highly anticipated DjangoCon US 2024 Tutorials! 🐍
+
+🗓️ Date: Sunday, September 22
+
+📍 Locations: Durham Convention CenterDurham, NC
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
📍 Location: [Durham Convention Center, Durham, NC](/venue/)
```

File: src/_content/posts/tutorials.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### 8:00 am to 6:00 pm
+
+In front of Junior Ballroom Registration
+
+### 9:30 am - 1:00 pm
+
+#### Tutorial Track A
+
+Led by Kuldeep Pisda Description: Efficient Django: Mastering Asynchronous Tasks with Celery, RabbitMQ, and Redis
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]: Maybe a better format might be `[tutorial title] led by [presenter name]`. Then link the tutorials to the actual talks.

File: src/_content/posts/tutorials.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+🗓️ Date: Sunday, September 22
+
+📍 Locations: Durham Convention CenterDurham, NC
+
+Gear up to boost your Django skills and elevate your web development expertise with a wide range of in-depth tutorials led by industry professionals. Whether you're an experienced Django developer or just beginning your journey, DjangoCon US 2024 offers something for everyone.
+Here’s a sneak peek of what’s in store:
+
+### 8:00 am - 9:00 am
+
+Grand Ballroom I/II Continental Breakfast
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
Continental Breakfast in Grand Ballroom I/II 
```

File: src/_content/posts/tutorials.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### 8:00 am to 6:00 pm
+
+In front of Junior Ballroom Registration
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
Registration in front of Junior Ballroom 
```

File: src/_content/posts/tutorials.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+Led by Melanie Pisda Description: pytest or Bust: Converting Your Django Tests to pytest
+
+#### Tutorial Track C
+
+Led by Felipe de Morais Description: Building Your First API with Django and Django Rest Framework
+
+### 12:30 pm - 1:30 pm
+
+Grand Ballroom I/IILunch
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
Lunch in Grand Ballroom I/II
```

File: src/_content/posts/tutorials.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+#### Tutorial Track C
+
+Led by Meagen Voss Description: How To Build More Accessible Websites with Wagtail 
+
+🎉 This is your chance to: Enhance your Django expertise, discover the latest best practices, connect with fellow developers, and gain insights from top industry experts.
+
+🚀 Don’t miss out on this incredible opportunity to grow your skills and network with the Django community!
+For more information and updates, visit our [website.](https://2024.djangocon.us)
+
+Tutorials are a separate registration and are $199 per session. Buy tutorial tickets before they sell out! [Buy your ticket](https://ti.to/defna/djangocon-us-2024) today
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
Tutorials are a separate registration and are $199 per session. Buy tutorial tickets before they sell out! [Buy your ticket](https://ti.to/defna/djangocon-us-2024) today.
```

File: src/_content/posts/tutorials.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+Led by Daniele Procida Description: Getting documentation done, now 
+
+#### Tutorial Track C
+
+Led by Meagen Voss Description: How To Build More Accessible Websites with Wagtail 
+
+🎉 This is your chance to: Enhance your Django expertise, discover the latest best practices, connect with fellow developers, and gain insights from top industry experts.
+
+🚀 Don’t miss out on this incredible opportunity to grow your skills and network with the Django community!
+For more information and updates, visit our [website.](https://2024.djangocon.us)
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]: I'm not sure about this line since they are presumably already on the site.

**Review by tim-schilling — COMMENTED** [review]
Thanks for putting this together @LideviK! What do you think about adding a little more context or personality to each of the tutorials as a way to boost attention/interest?

File: src/_content/posts/tutorials.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+Led by Lee Trout Description: Django DevOps 
+
+#### Tutorial Track B
+
+Led by Daniele Procida Description: Getting documentation done, now 
+
+#### Tutorial Track C
+
+Led by Meagen Voss Description: How To Build More Accessible Websites with Wagtail
```
Comment (tim-schilling) [inline src/_content/posts/tutorials.md]:
```suggestion
Led by Meagen Voss and Scott Cranfill Description: How To Build More Accessible Websites with Wagtail 
```

Comment (LideviK) [reply src/_content/posts/tutorials.md]: Yeah, I can do that 😊

**Comment by tim-schilling** [comment]
Screenshot from 2024-08-28 07-49-01

I'm not sure how everyone feels with the "we've got..." repetition. I decided to try something different

File: src/_content/posts/tutorials.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+#### Tutorial Track C
+
+Led by Felipe de Morais Description: Building Your First API with Django and Django Rest Framework
+
+### 12:30 pm - 1:30 pm
+
+Lunch in Grand Ballroom I/II
+
+### 30 pm to 5:00 pm
```
Comment (drewbrew) [inline src/_content/posts/tutorials.md]:
```suggestion
### 1:30 pm to 5:00 pm
```

File: src/_content/posts/tutorials.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+Led by Lee Trout Description: Django DevOps 
+
+#### Tutorial Track B
+
+Led by Daniele Procida Description: Getting documentation done, now 
+
+#### Tutorial Track C
+
+Led by Meagen Voss and Scott Cranfill Description: How To Build More Accessible Websites with Wagtail 
+
```
Comment (drewbrew) [inline src/_content/posts/tutorials.md:28]:
```suggestion

## Summary

```

File: src/_content/posts/tutorials.md (hunk 2 above)
Comment (drewbrew) [reply src/_content/posts/tutorials.md]: yeah, this feels awkward as it is

**Comment by tim-schilling** [comment]
@drewbrew I didn't push the changes

Edit: I did now

**Pull request merged**